src/state/hooks/RouterHooks.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { useEffect } from 'react';
```

Comment:
the /state/hooks directory is intended to contains only hooks relative to store.
This file should me move to a new directory like /hooks.